metrics/test_driver/util/file_utils.dart

```diff
@@ -102,28 +38,22 @@ class FileUtils {
     }
   }
 
-  /// Saves the process outputs from the [stderr] and [stdout]
-  /// to [fileName] in [workingDirPath] directory.
-  static void saveOutputsToFile(
-    Stream<List<int>> stdout,
-    Stream<List<int>> stderr,
+  /// Creates the empty [fileName].log file in [workingDirPath].
+  ///
+  /// If the file is already exists - cleans the file.
+  static File createLogFile(
```

Comment:
Maybe we make it generic, just createFile? With recreate (or cleanContent) to clear the file contents?

Reply:
We should probably store log file names with ".log" right away.